wordpress-mobile/gutenberg-mobile
# Update mobile code for changes in color settings/attributes

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/21257

To test:
 - Start the demo app
 - Make sure post open
 - Copy blocks from the web where custom colors are set
 - For example paragraph blocks:

```

<p style="--wp--color--text:#9bc2d3" class="has-text-color">text</p>

```

Check that after switching from HTML to visual the settings on the block are kept.

PR submission checklist:
